messages/de.json

```diff
@@ -1,4 +1,5 @@
 {
+  "Enter obligation comment": "Verpflichtungskommentar eingeben",
```

Comment:
Move this into "default" (same with other files)

Reply:
@rudra-superrr What is the status of this one ?